### PR TITLE
WEBAPP-214: Add poi functionality

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/routes/pois/containers/PoiPage.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        </>
+      } else {
+        const error = new ContentNotFoundError({type: 'poi', id: poiId, city, language})
+        return <FailureSwitcher error={error} />
+      }
+    }
+    return <>
+      <Helmet title={`${t('pageTitle')} - ${CityModel.findCityName(cities, city)}`} />
+      <Caption title={t('pois')} />
+      <List noItemsMessage={t('noPois')} items={pois} renderItem={this.renderPoiListItem} />
```
Comment (Schedulaar) [inline src/routes/pois/containers/PoiPage.js]: Maybe we want to apply some sorting (alphabetical?) to the pois?

File: src/routes/pois/components/PoiListItem.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+type PropsType = {|
+  poi: PoiModel
+|}
+
+class PoiListItem extends React.Component<PropsType> {
+  render () {
+    const {poi} = this.props
+    return (
+      <ListItem key={poi.path}
+                thumbnail={poi.thumbnail}
```
Comment (Schedulaar) [inline src/routes/pois/components/PoiListItem.js]: How does it react, if some pois have a thumbnail and some don't?
Are the items unaligned then?

File: src/routes/pois/components/PoiListItem.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                title={poi.title}
+                path={poi.path}>
+        <div>{poi.location.location}</div>
```
Comment (Schedulaar) [inline src/routes/pois/components/PoiListItem.js]: I think we should also add the excerpt here, as we do for the events. TBD in #feature-poi

File: src/routes/pois/containers/PoiPage.js (hunk 1 above)
Comment (klinzo) [reply src/routes/pois/containers/PoiPage.js]: Sounds reasonable :+1:

File: src/routes/pois/components/PoiListItem.js (hunk 3 above)
Comment (klinzo) [reply src/routes/pois/components/PoiListItem.js]: I don't know. Probably most pois are just autogenerated event locations which doesn't have an excerpt. We could make it optional of course. @maxammann what do you think?

Comment (maxammann) [reply src/routes/pois/components/PoiListItem.js]: excerpt and description are optional for events. @svenseeberg is this correct?

File: src/modules/app/containers/__tests__/Switcher.spec.js [hunk 4]
```diff
@@ -284,6 +307,14 @@ describe('Switcher', () => {
 
       expect(switcher).toMatchSnapshot()
     })
+
+    it('is the pois route', () => {
```
Comment (maxammann) [inline src/modules/app/containers/__tests__/Switcher.spec.js]: should be the pois route

File: src/routes/pois/components/PoiListItem.js (hunk 3 above)
Comment (klinzo) [reply src/routes/pois/components/PoiListItem.js]: I'll just leave it for now, we can add it later if an excerpt is provided.

File: src/modules/common/components/Page.js [hunk 5]
```diff
@@ -24,7 +24,7 @@ type PropsType = {|
   language: string,
   onInternalLinkClick: string => void,
   hijackRegExp?: RegExp,
-  children?: Array<React.Node>
+  children?: Array<React.Node> | React.Node
```
[the remark this answers is not in the document]
Comment (klinzo) [reply src/modules/common/components/Page.js]: Right :ok_hand:

File: src/modules/app/containers/__tests__/Switcher.spec.js (hunk 4 above)
Comment (klinzo) [reply src/modules/app/containers/__tests__/Switcher.spec.js]: Nope, this is good, the `describe()` says `should get the right page if data has been fetched and`.

**Pull request merged**